Cassini/app/src/main/java/org/x/cassini/MainActivity.java

```diff
@@ -2,13 +2,133 @@
+                tagsAct.putExtras(bundle);
+            }
+        });
+
+        //starred
+        starred = (TextView) findViewById(R.id.main_starred);
+        starred.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v){
+                Bundle bundle = new Bundle();
+                bundle.putString("starred", (String)starred.getText());
+                Intent starredAct = new Intent(getApplication(),StarredActivity.class);
+                starredAct.putExtras(bundle);
+            }
+        });
+
+        //stories
+        stories = (TextView) findViewById(R.id.main_stories);
+        stories.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v){
+                Bundle bundle = new Bundle();
+                bundle.putString("stories", (String)stories.getText());
+                Intent storiesAct = new Intent(getApplication(),StoriesActivity.class);
+                storiesAct.putExtras(bundle);
+            }
+        });
+
+        //settings
+        settings = (TextView) findViewById(R.id.main_all_entries);
+        settings.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v){
+                Bundle bundle = new Bundle();
+                bundle.putString("settings", (String)settings.getText());
+                Intent settingsAct = new Intent(getApplication(),SettingsActivity.class);
+                settingsAct.putExtras(bundle);
+            }
+        });
+
+        //new Entry
+        newEntry = (TextView) findViewById(R.id.main_all_entries);
+        newEntry.setOnClickListener(new View.OnClickListener(){
+            @Override
+            public void onClick(View v){
+                Bundle bundle = new Bundle();
+                bundle.putString("newEntry", (String)newEntry.getText());
+                Intent newEntryAct = new Intent(getApplication(),NewEntryActivity.class);
+                newEntryAct.putExtras(bundle);
+            }
+        });
+    }
+
+    private void initImageView(){
+        allEntriesIcon = (ImageView) findViewById(R.id.main_all_entries_icon);
+        timelineViewIcon = (ImageView) findViewById(R.id.main_timeline_view_icon);
+        tagsIcon = (ImageView) findViewById(R.id.main_tags_icon);
+        starredIcon = (ImageView) findViewById(R.id.main_starred_icon);
+        storiesIcon = (ImageView) findViewById(R.id.main_stories_icon);
+        settingsIcon = (ImageView) findViewById(R.id.main_settings);
+        goTo1 = (ImageView) findViewById(R.id.main_goTo1);
+        goTo2 = (ImageView) findViewById(R.id.main_goTo2);
+        goTo3 = (ImageView) findViewById(R.id.main_goTo3);
+        goTo4 = (ImageView) findViewById(R.id.main_goTo4);
+        goTo5 = (ImageView) findViewById(R.id.main_goTo5);
+        goTo6 = (ImageView) findViewById(R.id.main_goTo6);
```

Comment:
These findViewById calls are not necessary if you are not going to use those 'found' components anywhere in your code.